src/components/Landing/skills/skills.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import styles from './Skills.module.scss';
+
+const Skills = ({ skills }) => (
+  <div className={styles['container']}>
+    <div className={styles['text']}>
+      <h3>Notre expertise</h3>
+      <p>pour rester à la pointe de la technologie</p>
+    </div>
+    {displaySkills(skills)}
+  </div>
+);
+
+const displaySkills = (skills: any[]) => {
+  const render = [];
+  skills.forEach((skill, i) => {
+    const angle = (i * (360 / skills.length)) * (Math.PI / 180);
+    render.push(<GatsbyImage image={skill.node.childImageSharp.gatsbyImageData} alt="aws" className={styles['logo']}
```

Comment:
As mentioned in another comment, the use of a mapping function is good. Nonetheless, you should have a dynamiac `alt` attribute by saving it as static data. Also, add the `width` and `height` attributes.